Subsystems/Electronics/Testing/ElectronicLoad/Software/src/main.cpp

```diff
@@ -1,7 +1,76 @@
 #include <mbed.h>
 
+#define HIGH_SHORT (0.0008)
+#define HIGH_ON (0.5)
+#define MED_SHORT (0.0008)
+#define MED_ON (0.05)
+#define LOW_ON (0.005)
+#define GAIN (48 / 3.3)
 
-int main(void) {
+/* Dates:
+09/08/2019 - Set up repositories and libraries
+09/15/2019 - wrote description, outlined variables, outline pins
+*/
+AnalogIn  currentIn(INPUT_CURRENT);       // ADC
+AnalogIn  currentOut(OUTPUT_CURRENT);     // ADC
+AnalogIn  voltageIn(INPUT_VOLTAGE);       // ADC
+AnalogIn  voltageOut(OUTPUT_VOLTAGE);     // ADC
+AnalogOut currentSet(OUTPUT_CURRENT_SET); // DAC
+
+Serial serial(USBTX, USBRX); // UART1 & 2
+
+DigitalOut senseHigh(SENSE_RANGE_LOW); // 0.5 OHM 100 mA = .005W
+DigitalOut senseMid(SENSE_RANGE_MED);  // 50 mOHM 1 A = 0.05W
+// Shunt-High is 5mOHM and 100 A = 0.5W
+
+/*
+ * Description: To write a program that can run the Electronic Load hardware.
+ * All variables including but not limited to: A variable knob, Lcd screen, four
+ * connectors.
+ *
+ * Connectors are:
+ * Current Conductor - (takes current and sends current back to itself),
+ * Input Current Connector - Same as Current Sensor (connects to power
+ * transistor via current), UART Connector - accepts signal through
+ * microcontroller - !peripheral to the microcontroller! Fan - is thermally
+ * connected to power transistor and is controlled by microcontroller via PWM.
+ * Voltage Sense Connector - accepts two sources of voltage into itself.
+ *
+ * Will need at least three outputs to turn on and off the three shunt
+ * resistors.
+ */
+/* Encoder has 2 random pins and buttons have random pins */
 
+void setCurrent(double current) {
```

Comment:
Add doxygen function header to this (type `/**` [ENTER]). Also want to return a result if the current is outside of the proper range?